sql/clients_monthly_v1.init.sql

```diff
@@ -0,0 +1,18 @@
+CREATE IF NOT EXISTS TABLE
+  clients_monthly_v1
+PARTITION BY
+  submission_date AS
+SELECT
+  DATE(NULL) AS submission_date,
+  CURRENT_DATETIME AS generated_time,
+  client_id,
+  sample_id,
+  ARRAY(
+  SELECT
+    AS STRUCT clients_daily_v6.* EXCEPT (client_id,
+      sample_id)) AS days
```

Comment:
This seems nice to factor out the fields that are guaranteed to be static.

Reply:
yeah, I was trying to think of any others, but nothing appeared to.

we could also add a `days_since_seen` or `last_seen_date` column for clustering to make it easier and cheaper to filter rows from the previous day's output that won't be used in today's output

Reply:
My gut says that would be premature optimization at this point.